packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -104,28 +104,25 @@ class CodeSnippetDisplay extends MetadataDisplay<
     this._evtMouseUp = this._evtMouseUp.bind(this);
   }
 
-  // Handle code snippet insert into an editor
+  // Handle code snippet insertion into an editor
   private insertCodeSnippet = async (snippet: IMetadata): Promise<void> => {
-    const widget: Widget = this.props.getCurrentWidget();
-    const snippetStr: string = snippet.metadata.code.join('\n');
+    const widget = this.props.getCurrentWidget();
+    const snippetStr = snippet.metadata.code.join('\n');
```

Comment:
is there a reason to make these less specific?

Reply:
As long as there are inferred as `Widget` and `string` is there any reason for the annotation?

Reply:
Less verbose. And to follow the convention across the rest of the code, unless we really need to specify a type on variable declaration, like when calling some function that can return any type

Reply:
it's just a readability thing, I'm surprised this got past the linter though, usually it complains when things aren't typed. This isn't blocking or anything, just seemed like a code change for "no reason"

Reply:
If we want to type our variables, should we consider doing it consistently across all our code and packages?

Reply:
I thought the linter was already enforcing typed vars across our code base but it seem like it is not the case (unsure if this was changed or I just mis-implemented it)

Reply:
I think it only enforces it in areas that are more bug prone, like the return type of a function.

The typescript eslint plugin recommends using the rule `@typescript-eslint/no-inferrable-types`

Reply:
either way, if the linter doesn't mind this then I'm ok either way for now

Reply:
@ajbozarth you might want to take a look at https://github.com/elyra-ai/eslint-config-elyra, I'm trying to put together a shared eslint config if you have an rule suggestions

Reply:
I'll try to take a look, but why are we changing our lint rules exactly? I spent quite a while creating our current ones to match what the jupyter ecosystem uses. Or is that repo just moving the current lint rules into an external repo for multiple repos to share?

Reply:
the intention is to move current lint rules to external repo for all repos to share, but I'm using what I currently have set up for `pipeline-editor` so I'm not sure if it matches up 100% with what is in `elyra`

Reply:
the intention is to move current lint rules to external repo for all repos to share, but I'm using what I currently have set up for `pipeline-editor` so I'm not sure if it matches up 100% with what is in `elyra`

Reply:
We should be consistent, but the "implementation details" are still being discussed and we should definitely use the one from Elyra as the base one.